CheckDrive.Web/CheckDrive.Web/Views/Dashboard/Index.cshtml

```diff
@@ -103,27 +103,21 @@
         </div>
     </div>
 </div>
-
 <div class="row">
     <div class="col">
-        <div class="widget">
-            <div class="p-4">
-                <h5 class="fw-bold">Qarzlar</h5>
-            </div>
-            <div class="px-4 pb-4">
+            <div style="margin-bottom:10px;">
                 <ejs-grid id="resent-debts" allowPaging="true"
```

Comment:
Since there are no debts in the current situation, the id should also be changed from resent-debts to resent-checkpoints